libcloud/common/exceptions.py

```diff
@@ -47,7 +51,7 @@ class RateLimitReachedError(BaseHTTPError):
     message = '%s Rate limit exceeded' % (code)
 
     def __init__(self, *args, **kwargs):
```

Comment:
Can we make sure that `code`, `message` and `headers` attributes are set on `RateLimitReachedError` ?  The fact that `RateLimitReachedError` extends `BaseHTTPError` but doesn't set these fields has caused real problems for us in our exception handlers in the past.

Reply:
Sure, will add a superclass constructor call to it.